deps/rabbit/src/rabbit_stream_queue.erl

```diff
@@ -96,8 +96,7 @@
                         soft_limit :: non_neg_integer(),
                         slow = false :: boolean(),
                         readers = #{} :: #{rabbit_types:ctag() => #stream{}},
-                        writer_id :: binary(),
-                        filtering_supported :: boolean()
```

Comment:
Are we 100% sure that this record is never stored in Mnesia/Khepri and never sent to another node?

Reply:
yes